packages/api-client/tests/test-executions.js

```diff
@@ -37,7 +37,8 @@ test('getExecutions calls the callback with the expected object', async (t) => {
     payload: {
       httpMethod: 'GET',
       resource: '/{proxy+}',
-      path: '/executions'
+      path: '/executions',
+      queryStringParameters: undefined
```

Comment:
Why can't we just exclude `queryStringParameters` if we're wanting it to be `undefined`?

Reply:
The test will holler at you, since that is the expected value